src/ReadSettings.jl

```diff
@@ -0,0 +1,70 @@
+using YAML
+
+data_type_dict = Dict(
+    "int" => Int64,
+    "float" => Float64,
+    "str" => String,
+    "bool" => Bool,
+    "elas" => Vector{Vector{Float64}},
+    "date" => String,
+)
+
+function classify_dict(dict::Dict{Any, Any})
```

Comment:
I'm struggling a bit to understand what this function does. A docstring might help.